modules/distr-tracing-install-jaeger-operator.adoc

```diff
@@ -5,48 +5,45 @@ This module included in the following assemblies:
 
 :_mod-docs-content-type: PROCEDURE
 [id="distr-tracing-jaeger-operator-install_{context}"]
-= Installing the {JaegerName} Operator
+= Installing the {JaegerOperator} Operator
 
-To install {JaegerName}, you use the link:https://operatorhub.io/[OperatorHub] to install the {JaegerName} Operator.
+You can install the {JaegerOperator} Operator through the link:https://operatorhub.io/[*OperatorHub*].
 
 By default, the Operator is installed in the `openshift-operators` project.
 
 .Prerequisites
 * You have access to the {product-title} web console.
 * You have access to the cluster as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
-* If you require persistent storage, you must also install the OpenShift Elasticsearch Operator before installing the {JaegerName} Operator.
-
-[WARNING]
-====
-Do not install Community versions of the Operators. Community Operators are not supported.
-====
+* If you require persistent storage, you must install the {es-op} before installing the {JaegerOperator} Operator.
 
 .Procedure
 
 . Log in to the {product-title} web console as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
 
 . Navigate to *Operators* -> *OperatorHub*.
 
-. Type *distributed tracing platform* into the filter to locate the {JaegerName} Operator.
+. Search for the {JaegerOperator} Operator by entering *distributed tracing platform* in the search field.
 
-. Click the *{JaegerName} Operator* provided by Red Hat to display information about the Operator.
+. Select the *{JaegerOperator}* Operator, which is *provided by Red Hat*, to display information about the Operator.
```

Comment:
If `provided by Red Hat` is part of the Gui I'd provide a noun after it such as page or tab.

Reply:
Thank you. It's not part of the GUI.